src/components/v2v3/V2V3Project/ProjectActivity/eventElems/SetFundAccessConstraintsEventElem.tsx

```diff
@@ -33,7 +24,9 @@ export default function SetFundAccessConstraintsEventElem({
           ) : (
             <AmountInCurrency
               amount={event.distributionLimit}
-              currency={V2V3CurrencyName(event.distributionLimitCurrency)}
+              currency={V2V3CurrencyName(
+                event.distributionLimitCurrency as V2V3CurrencyOption,
```

Comment:
ok Apollo also doesn't type `V2V3CurrencyOption`, but this is infrequently used and i don't mind using `as`. Otherwise we could update the pv override script to also override this property

Reply:
lgtm! lfg